tools/webpack.config.ts

```diff
@@ -78,6 +78,16 @@ export default (env?: string): webpack.Configuration => {
                         },
                     ],
                 },
+                {
+                    test: /\.handlebars$/,
+                    loader: 'handlebars-loader',
+                    options: {
+                        // Tell webpack not to explicitly require these.
+                        knownHelpers: ['if', 'unless', 'each', 'with',
+                            'partial', 'plural', 'eq', 'and', 'or', 'not',
+                            't', 'tr'],
```

Comment:
What's the story with this list?  Does it need to be synchronized with some list elsewhere in the codebase?

Reply:
These are the helpers defined in templates.js combined with the ones we originally passed to handlebars in compile-handlebar-templates line 33. This is to let webpack know these helpers are defined at run time and don't try to require them when bundling. Nothing will happen if these list is not correct, handlebar-loader just have to do more guessing work.

Reply:
OK.